release-notes/unreleased/IGAPP-336-Progress-Circle.yml

```
@@ -0,0 +1,7 @@
+issue_key: IGAPP-336
+show_in_stores: true
+platforms:
+  - ios
+  - android
+en: Now a progress circle is shown when loading content
+de: Beim Laden wird jetzt der Fortschritt in Form eines Kreises angezeigt
```

Maybe:
```suggestion
en: We added a nice animation to show you the current loading progress.
de: Die App gibt dir nun Feedback über den Fortschritt beim Ladevorgang.
```
or similar